src/records/Course.js

```diff
@@ -0,0 +1,26 @@
+import {Record} from 'immutable';
+
+export default class Course extends Record({
+  id: null,
+  name: null,
+  section: null,
+  descriptionHeading: null,
+  description: null,
+  room: null,
+  ownerId: null,
+  creationTime: null,
+  updateTime: null,
+  enrollmentCode: null,
+  courseState: null,
+  alternateLink: null,
+  teacherGroupEmail: null,
+  courseGroupEmail: null,
+  guardiansEnabled: null,
+  calendarId: null,
```

Comment:
Include most of the info from the assignment that comes back in the payload except for these two. Give that we are only using a couple of these items I am wondering if it makes sense to store all of these.
```
"teacherFolder": {
    object(DriveFolder)
  },
  "courseMaterialSets": [
    {
      object(CourseMaterialSet)
    }
  ],
```

Reply:
I’d be inclined to stick with just the information we’re using—no real cost to adding more properties later if we need them.